File: 136-single-number/single-number.go
func singleNumber(nums []int) int {
 
 maps := make(map[int]int)

 for _,freq :=range nums{
    maps[freq] +=1
 }   

 for index,freq := range maps{
    if freq ==1{
        return index
    }
 }
 return 0
}

File: 136-single-number/single-number.py
class Solution:
    def singleNumber(self, nums: List[int]) -> int:
        res = 0
        freq_dict = {}
        for i in nums:
            if i in freq_dict.keys():
                freq_dict[i] +=1
            else:
                freq_dict[i] = 1
        for k,v in freq_dict.items():
            if v == 1:
                res = k
        return res